src/views/network/Network.vue

```diff
@@ -1,92 +1,109 @@
 <template>
-    <splitpanes class="network-panel default-theme fill-height">
-        <pane style="min-width: 250px" :size="100 - detail_size">
-            <!--suppress HtmlUnknownAttribute -->
-            <div ref="scroll" class="network-container absolute-expand overflow-y-auto">
-                <v-list v-if="session_list.length" dense>
-                    <v-list-item-group v-model="selected" color="primary">
-                        <div v-for="(s,i) in session_list" :key="i">
-                            <div class="text-center text--text font-weight-bold pt-4">{{ formatTimestamp(s.timestamp, true) }}</div>
-                            <v-list-item class="request-item" two-line v-for="uid in s.requests" :key="uid">
-                                <v-list-item-content class="py-0">
-                                    <v-list-item-title class="pt-2">
-                                        <span class="method" :class="requests[uid].headers.method.toLowerCase()">{{ requests[uid].headers.method }}</span>
-                                        {{ requests[uid].headers.url.pathname }}
-                                    </v-list-item-title>
-                                    <v-list-item-subtitle class="origin pb-2">{{ requests[uid].headers.url.origin }}</v-list-item-subtitle>
-                                </v-list-item-content>
-                                <div class="mr-3">
-                                    <table>
-                                        <tr class="request-info">
-                                            <td class="request-status" :class="statusColor(requests[uid].status)">
-                                                <span v-if="requests[uid].status" class="font-weight-bold">{{ requests[uid].status }}</span>
-                                                <v-icon v-else class="request-loading">mdi-timelapse</v-icon>
-                                            </td>
-                                            <td class="request-timestamp font-weight-bold text-right">{{ formatTimestamp(requests[uid].timestamp) }}</td>
-                                        </tr>
-                                        <tr v-if="requests[uid].response" class="v-list-item__subtitle request-info">
-                                            <td>{{ formatDuration(requests[uid].response.timestamp - requests[uid].timestamp) }}</td>
-                                            <td class="text-right">{{ filesize(requests[uid].response.body.size) }}</td>
-                                        </tr>
-                                    </table>
+    <pane>
```

Comment:
You cannot have a `<pane>` outside of `<splitpanes>`, there's a huge warn in the console.
I also prefer to embed this search bar in the request list `<pane>` instead of being "global", because it has nothing to do with the request details.